### DISCUSSION TITLE
Linker error when using libktx

### DISCUSSION THREAD

**Discussion by ChunkTreasure1** [KhronosGroup/KTX-Software#530] [Q&A] [answered]
Hello! I am trying to use libktx, I have forked the project and build it (statically), I linked the ktx.lib and added the include directory to my includes, however I get a linker error when trying to use ktxTexture_CreateFromNamedFile. Am I linking the right library?

**Comment by MarkCallow** [comment, marked as answer]
I am guessing, given lack of info about platform and actual errors ...

If you are building on Windows and the errors are unresolved symbols of the form `__imp_*` then to solve it you need to specify -DKHRONOS_STATIC on the compile command(s) when building your application. This will give your application the correct linkage for linking to a static library.

**Reply by MarkCallow** [reply]
Given lack of further response from the OP I'm assuming my answer fixed the problem so marking as answer.
